app/models/vm/operations.rb

```diff
@@ -6,17 +6,23 @@ module Vm::Operations
   include_concern 'Lifecycle'
 
   included do
-    supports :html5_console do
-      consup = %w[vnc webmks spice].any? { |type| send(:console_supported?, type) }
-      unsupported_reason_add(:html5_console, _("The web-based HTML5 Console is not supported")) unless consup
-    end
+    # TODO: convert to supports_not :spice_console
+    supports(:spice_console) { unsupported_reason_add(:spice_console, N_("Console not supported")) unless console_supported?('spice') }
+    supports(:vnc_console) { unsupported_reason_add(:vnc_console, N_("Console not supported")) unless console_supported?('vnc') }
+    supports(:native_console) { unsupported_reason_add(:native_console, _("VM NATIVE Console not supported")) unless console_supported?('NATIVE') }
+    supports(:vmrc_console) { unsupported_reason_add(:vmrc_console, _("VMRC Console not supported")) unless console_supported?('VMRC') }
+    supports(:webmks_console) { unsupported_reason_add(:webmks_console, N_("Console not supported")) unless console_supported?('webmks') }
 
-    supports :vmrc_console do
-      unsupported_reason_add(:vmrc_console, _("VMRC Console not supported")) unless console_supported?('VMRC')
+    supports :console do
+      unless supports?(:spice_console) || supports?(:vnc_console)
```

Comment:
Realize this was pulled from https://github.com/ManageIQ/manageiq/pull/21990/files#diff-1cd4edb18c242ba5e1f5795da3f815cde970e5e9672cdd7d96b1a101fe7c465fL1679 but I wonder why `webmks` isn't an option here

Reply:
I was doing it blind but I think that is a good idea

Reply:
TODO: move to actual `supports :console` in child providers

Reply:
this changes vmware cloud manager and no others